src/SessionSetup/OrbitServiceInstance.cpp

```diff
@@ -82,7 +82,8 @@ ErrorMessageOr<void> OrbitServiceInstanceImpl::Shutdown() {
 
   bool wait_finished_result = process_.waitForFinished(kWaitTimeInMilliSeconds);
   if (!wait_finished_result) {
-    return ErrorMessage{"The OrbitService shutdown attempt failed."};
+    return ErrorMessage{absl::StrFormat("Shutting down OrbitService timed out after %dms.",
```

Comment:
```suggestion
    return ErrorMessage{absl::StrFormat("Shutting down OrbitService timed out after %d ms.",
```

Nit: I would prefer a space between the number and the unit.